.github/ISSUE_TEMPLATE/add_dataset.yml

```diff
@@ -0,0 +1,79 @@
+name: Add dataset
+description: Suggest a new Library, Archive or Museum dataset to include in the Hugging Face Hub.
+title: "[Add Dataset]: "
+labels: ["dataset"]
+assignees: "davanstrien"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for suggesting a dataset!
+  - type: textarea
+    id: data-description
+    attributes:
+      label: Dataset description
+      description: A brief description of the dataset and why it might be of interest
+      placeholder: Please summarise the dataset and why it might be interesting to add to the hub. 
+      value: "This dataset is awesome! Here is why..."
+    validations:
+      required: true
+  - type: dropdown
+    id:  dataset-modality
+    attributes:
+      label: Dataset modality
+      description: What is the modality of the data?
+      options:
+        - Text
+        - Image
+        - Audio
+        - Video
+        - Mixed
+        - Other
+    validations:
+      required: true
+  - type: checkboxes
+    id: type-of-dataset
+    attributes:
+      label: Type of dataset
+      description: What type of datset is this?
+      options:
+        - label: A large unlabeled text corpus, i.e. something that could be used for training language models
+        - label: A large unlabeled image corpus, i.e. something that could be used for pre-training image models
+        - label: An annotated/labelled dataset, i.e. somethng that could be used to train/evaluate a machine learning model
+        - label: A benchmark dataset, i.e. a dataset that has been used to evaluate the performance of machine learning models
+        - label: Other
+    validations:
+      required: true
+  - type: input
+    id: licence
+    attributes:
+      label: Dataset licence
+      description: Please include the license for the dataset here (either include the licence or a link) 
```

Comment:
I'm thinking maybe it could be more useful having a dropdown, like we have when adding a dataset on the Hub: click inside the License field here https://huggingface.co/new-dataset

We have also a list in Datasets: https://github.com/huggingface/datasets/blob/master/src/datasets/utils/resources/licenses.json

Optionally, we could also have a textarea in case it has a specific/custom license: the user can put the text of the license or a link to it.

Reply:
Maybe just proposing some of the most usual licenses? So that it will be faster to find it...

Reply:
My main reason for doing this was to avoid a long dropdown menu.  Perhaps a solution is to include the most common ones as a dropdown (the various creative commons licenses will likely capture >80% of datasets) and have an input box for ones with more complicated licenses/multiple licenses. 

I can add the most common ones as a dropdown choice if that sounds okay?